### PR TITLE
Fix shrinking of icons in long folder name

### PR DESCRIPTION
**Pull request by skjnldsv**
1. Narrow window with long folder
2. See icon get smaller than 44px width

### CODE REVIEW

File: src/components/Navigation.vue [hunk 1]
```diff
@@ -191,6 +196,7 @@ $previous: 0;
 			}
 			&--root &__title {
 				padding-left: #{$marginW}px;
+				padding-left: #{$marginW}px;
```
Comment (Mikescops) [inline src/components/Navigation.vue]: Duplicate here?

Comment (skjnldsv) [inline src/components/Navigation.vue]:
```suggestion
```

Comment (skjnldsv) [reply src/components/Navigation.vue]: fixed

**Pull request merged**